src/lib/apmJson.ts

```diff
@@ -7,34 +7,26 @@ import {
 import log from 'electron-log';
 import { readJsonSync, writeJsonSync } from 'fs-extra';
 import path from 'path';
-
-/**
- * Returns the path of `apm.json`.
- *
- * @param {string} instPath - An installation path
- * @returns {string} The path of `apm.json`
- */
-function getPath(instPath) {
-  return path.join(instPath, 'apm.json');
-}
+import { ApmJsonObject } from '../types/apmJson';
+import { PackageItem } from '../types/packageItem';
 
 /**
  * Gets the object parsed from `apm.json`.
  *
  * @param {string} instPath - An installation path
  * @returns {object} An object like `package.json`
  */
-function getApmJson(instPath) {
+function getApmJson(instPath: string) {
```

Comment:
```suggestion
function getApmJson(instPath: string): ApmJsonObject {
```
Corresponding to the below.